Packs/FeedLOLBAS/pack_metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "LOLBAS Feed",
+    "description": "Fetches indicators from LOLBAS.",
```

Comment:
Please have Arik provide a better description, or take the one from the integration.